packages/realm-react/src/__tests__/RealmProvider.test.tsx

```diff
@@ -163,6 +175,45 @@ describe("RealmProvider", () => {
 
     expect(newSchemaNameContainer).toHaveTextContent("cat");
   });
+  describe("initially renders a fallback, until realm exists", () => {
+    it("as a component", async () => {
+      const App = () => {
+        return (
+          <RealmProvider fallback={<View testID="fallbackContainer" />}>
```

Comment:
You need to pass a function (or class) to properly test this:

```suggestion
          <RealmProvider fallback={() => <View testID="fallbackContainer" />}>
```